docs/root/faq/configuration/timeouts.rst

```diff
@@ -155,6 +155,15 @@ TCP
   is the amount of time that the TCP proxy will allow a connection to exist with no upstream
   or downstream activity. The default idle timeout if not otherwise specified is *1 hour*.
 
+* The TCP protocol :ref:`idle_timeout <envoy_v3_api_field_extensions.upstreams.tcp.v3.TcpProtocolOptions.idle_timeout>`
+  is defined in a :ref:`TcpProtocolOptions <envoy_v3_api_msg_extensions.upstreams.tcp.v3.TcpProtocolOptions>`
+  used by all TCP connections from pool. The idle timeout is the time at which
+  a upstream connection will be terminated if the connection is not in use. The timeout is guarded by
```

Comment:
`the connection is not in use`. I think it would be helpful to define here (as you did in other places in the docs) exactly what that means, as one reasonable interpretation of `not in use` is `no data being transferred in either direction`, but that is a covered by the tcpproxy idle timeout, not this one.